tom_alertstreams/alertstreams/hopskotch.py

```diff
@@ -114,7 +114,7 @@ def listen(self):
                             # First check all wildcard topics to see if they will match this topic
                             matched_handler = False
                             for topic in self.alert_handler.keys():
-                                if '*' in topic and re.match(topic, metadata.topic):
+                                if topic != '*' and '*' in topic and re.match(topic, metadata.topic):
```

Comment:
'*' isn't a valid regex expression, but I like its simplicity for configuration, so just don't use it as a regex expression and things should be fine.